solution-base/build.sh

```diff
@@ -69,42 +67,46 @@ onessl_found() {
     if [ -x "$(command -v onessl)" ]; then
         onessl version --check=">=${ONESSL_VERSION}" >/dev/null 2>&1 || {
             # old version of onessl found
-            echo "Found outdated onessl"
+            echo found outdated onessl
             return 1
         }
-        export ONESSL=onessl
+        echo onessl found
         return 0
     fi
+    echo onessl not found
     return 1
 }
 
 function gen_certs()
 {
     echo generating certs
     onessl create ca-cert
-    onessl create server-cert server --domains=kubedb-$KUBEDB_OPERATOR_NAME.$KUBEDB_NAMESPACE.svc
+    onessl create server-cert server --domains=kubedb-$KUBEDB_OPERATOR_NAME.svc
     export SERVICE_SERVING_CERT_CA=$(cat ca.crt | onessl base64)
     export TLS_SERVING_CERT=$(cat server.crt | onessl base64)
     export TLS_SERVING_KEY=$(cat server.key | onessl base64)
```

Comment:
I might be wrong, but it might be unsafe to build the certs during packaging stage. The ISO will have a cert associated with it, and the cert can possible be used in multiple deployments

Reply:
I did think it was a little strange to package a cert this way but at the same time wasn't sure if it was insecure or an alternative if it was.

Reply:
Yes, certs should not be packaged. Alternatives would be to generate them into a `Secret` by a `Job` at install time, or ship certificate resources to be handled by `cert-manager` (which we have to make sure is there anyway), and use that secret.

Reply:
@alexanderchan-scality Depends on the utility of the certificate, I don't know for what it's used but indeed, my guess would be to generate those certificates at installation time.

Also the following comment lacks a lot of product knowledge but, I believe we will use cert-manager, is it possible to create a cert-manager certificate object that creates it the way we want at install time? that way we just have need to expose a secret name instead of the certificate itself.

Reply:
@RachedBenMustapha updated to use cert-manager